src/content/self-hosted/helm-configuration.mdx

```diff
@@ -905,6 +887,39 @@ secret:
     your.custom.label: "20"
 ```
 
+### serviceAccounts
+
+Use the `serviceAccounts` section to configure labels, annotations and role bindings to be applied to each service account generated by Okteto for user accounts.
+
+- `annotations`: Annotations added to each service account generated by Okteto for user accounts. A common use case for adding service account annotations is to [get access via IAM Roles or Workload Identity](https://community.okteto.com/t/support-for-gke-workload-identities-for-okteto-self-hosted/39/4) to create Cloud resources as part of your Okteto Manifest `deploy` commands.
+- `labels`: Labels added to each service account generated by Okteto for user accounts.
+- `roleBindings.namespaces`: Okteto assigns this cluster role to every admin, owner and member of a namespace via a namespace-scoped role binding on every namespace managed by Okteto. This cluster role is **not** created by Okteto, it must exist in the cluster. Default value is `cluster-admin`.
+- `roleBindings.previews`: Okteto assigns this cluster role to every Okteto user via a namespace-scoped role binding on every **global** preview environment. This cluster role is **not** created by Okteto, it must exist in the cluster. Default value is `view`.
+- `clusterRoleBinding`: Okteto assings this cluster role to every user via a cluster role binding. By default, this behavior is disabled. This can be useful to give access to cluster level resources to every developer account, like accessing the Node API. This cluster role is **not** created by Okteto, it must exist in the cluster.
```

Comment:
```suggestion
- `clusterRoleBinding`: Okteto assigns this cluster role to every user via a cluster role binding. By default, this behavior is disabled. This can be useful to give access to cluster level resources to every developer account, like accessing the Node API. This cluster role is **not** created by Okteto, it must exist in the cluster.
```